packages/mux-player/src/template.ts

```diff
@@ -128,6 +128,7 @@ export const content = (props: MuxTemplateProps) => html`
       custom-domain="${props.customDomain ?? false}"
       src="${!!props.src ? props.src : props.playbackId ? toMuxVideoURL(props) : false}"
       cast-src="${!!props.src ? props.src : props.playbackId ? toMuxVideoURL(props) : false}"
+      cast-receiver="${!!props.castReceiver ?? false}"
```

Comment:
(blocking) remove double negation?

Reply:
Thanks for the catch! (dangers of flying blind with code changes 😅)